client/src/components/FieldTag/index.tsx

```diff
@@ -35,17 +35,20 @@ const FieldTag: FC<FieldTagProps> = ({
     return `${split.join('-')}-tag-color`;
   };
 
+  const getFullClassName = (): string => {
+    const storedBackground =
+      hexcode === undefined ? '' : toClassName(toString(content));
+
+    return classNames('interest-team-label', storedBackground, rest.className);
+  };
+
   return (
     <Label
       {...rest}
-      content={titleCase(toString(name !== '' ? name : content))}
+      content={name || titleCase(toString(content))}
```

Comment:
Should name not be made titleCase?

Reply:
I decided no because if Admin wants to type Interest or Team with a specific format we don't want to break the formatting. We will fix this everywhere as we go on